wordle/src/main/kotlin/wordle/domain/Tile.kt

```diff
@@ -0,0 +1,14 @@
+package wordle.domain
+
+enum class Tile(
+    val symbol: String,
+) {
+
+    GREEN("🟩"),
+    YELLOW("🟨"),
+    GRAY("⬜");
+
+    fun isGreen() = this == GREEN
+
+    fun isGray() = this == GRAY
```

Comment:
```suggestion
    val isGreen
        get() = this == GREEN

    val isGray
        get() = this == GRAY
```

요렇게 커스텀 게터를 사용하면 좀 더 코틀린스럽게 사용할 수 있어용

Reply:
오 이런식으로 활용할 수도 있네요 👍